backup-parameter-store/riff-raff.yaml

```diff
@@ -0,0 +1,25 @@
+stacks: [frontend]
+regions: [eu-west-1]
+
+deployments:
+  backup-parameter-store:
+    type: aws-lambda
+    parameters:
+      bucket: aws-frontend-artifacts
+      fileName: backup-parameter-store.jar
+      functions:
+        PROD:
+          name: frontend-PROD-backup-parameter-store-Lambda-7XXT59MIFU7D
+          filename: backup-parameter-store.jar
+        CODE:
+          name: frontend-CODE-backup-parameter-store-Lambda-FBLJ3W8PGXM9
+          filename: backup-parameter-store.jar
+
+  backup-parameter-store-cfn:
+    type: cloud-formation
+    app: backup-parameter-store
+    parameters:
+      prependStackToCloudFormationStackName: false
```

Comment:
Is it worth setting this to true? (Typically, we do include the stack name, although there are plenty of exceptions.)

Reply:
Stack name currently looks like this frontend-CODE-backup-parameter-store.
Is this okay?